src/routes/safe/components/Transactions/helpers/TxParametersDetail/index.tsx

```diff
@@ -99,43 +126,146 @@ export const TxParametersDetail = ({
           </StyledText>
 
           <TxParameterWrapper>
-            <ColoredText
-              size="lg"
-              isOutOfOrder={isTxNonceOutOfOrder}
-              color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-            >
+            <ColoredText size="lg" isOutOfOrder={isTxNonceOutOfOrder} color={getColor()}>
               Safe nonce
             </ColoredText>
-            <ColoredText
-              size="lg"
-              isOutOfOrder={isTxNonceOutOfOrder}
-              color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-            >
+            <ColoredText size="lg" isOutOfOrder={isTxNonceOutOfOrder} color={getColor()}>
               {txParameters.safeNonce}
             </ColoredText>
           </TxParameterWrapper>
 
           {showSafeTxGas && (
             <TxParameterWrapper>
-              <Text
-                size="lg"
-                color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-              >
+              <Text size="lg" color={getColor()}>
                 SafeTxGas
               </Text>
-              <Text
-                size="lg"
-                color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-              >
+              <Text size="lg" color={getColor()}>
                 {txParameters.safeTxGas}
               </Text>
             </TxParameterWrapper>
           )}
           <StyledButtonLink color="primary" textSize="xl" onClick={onEdit}>
             Edit
           </StyledButtonLink>
+          {storedTx?.length > 0 && <TxAdvancedParametersDetail tx={storedTx[0]} />}
         </AccordionDetailsWrapper>
       </AccordionDetails>
     </Accordion>
   )
 }
+
+const TxParameterEndWrapper = styled.span`
+  display: flex;
+  justify-content: flex-end;
+  gap: 4px; // EthHashInfo uses a gap between the address and copy button
```

Comment:
Not sure if the comment is needed

Reply:
I will leave it as it seems trivial enough to justify it imo.

Reply:
I think we should use the `src/theme/variables.js` values for spacing and paddings. wdyt?

Reply:
This relates to a fixed style within `EthHashInfo`, so I will leave this for now.